Preferences/DoublePreference.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  DoublePreference.swift
+//  Pods
+//
+//  Created by Filip Dolník on 25.05.15.
+//
+//
+
+import Foundation
+
+public class DoublePreference: Preference {
+    
+    public typealias T = Double
```

Comment:
I think that you don't need to specify the T directly. You just need to use `Double` instead of T everywhere. But I'll leave that up to you.